packages/apps/esm-login-app/src/footer.component.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { interpolateUrl, useConfig } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import { type ConfigSchema } from './config-schema';
+import styles from './login/login.scss';
+
+const Footer: React.FC = () => {
+  const {t} = useTranslation();
+  const config = useConfig<ConfigSchema>();
+  const logos = config.footer.logos || [];
+
+  return (
+    <div className={styles.footer}>
+      <p className={styles.poweredByTxt}>{t('poweredBy', 'Powered by')}</p>
+      <div className={styles.logosContainer}>
+        <svg role="img" className={styles.poweredByLogo}>
+          <use href="#omrs-logo-full-color"></use>
+        </svg>
+        {logos.map((logo, index) => (
+          <img
+            key={index}
+            alt={logo.alt ? t(logo.alt) : t('footerlogo', 'Footer Logo')}
+            className={styles.poweredByLogo}
+            src={interpolateUrl(logo.src)}
```

Comment:
```suggestion
            src={logo.src}
```

`interpolateUrl` does nothing with no second argument